.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -131,7 +246,7 @@ jobs:
   notify-slack-on-failure:
     runs-on: ubuntu-latest
     needs: [init-github-context, rawls-build-tag-publish-job, create-bee-workflow, rawls-swat-e2e-test-job, destroy-bee-workflow] # Want to notify regardless of which step fails
-    if: ${{ failure() }}
+    if: false
```

Comment:
I think we need to revert this, assuming this was forced to false for testing.

Reply:
Reverted